app/src/main/java/com/github/epfl/meili/map/PoiMarkerViewModel.kt

```diff
@@ -29,6 +30,11 @@ class PoiMarkerViewModel : ViewModel(), Observer, LocationListener {
     val mPointsOfInterestStatus: MutableLiveData<Map<String, PointOfInterestStatus>> =
             MutableLiveData(HashMap())
 
+    override fun onProviderEnabled(provider: String) {}
```

Comment:
Why were these functions added?

Reply:
To prevent this error: 
```
com.github.epfl.meili.map.MapActivityTest > b_locationButtonClickableAfterPermissionGrant[ONEPLUS A6000 - 10] FAILED 
        java.lang.AbstractMethodError: abstract method "void android.location.LocationListener.onProviderDisabled(java.lang.String)"
        at android.location.LocationManager$ListenerTransport._handleMessage(LocationManager.java:369)
Tests on ONEPLUS A6000 - 10 failed: Instrumentation run failed due to 'Process crashed.'
```

Reply:
Sounds good thanks!